pettingzoo/utils/conversions.py

```diff
@@ -311,7 +325,7 @@ def add_new_agent(self, new_agent):
         self.rewards[new_agent] = 0
         self._cumulative_rewards[new_agent] = 0
 
-    def step(self, action: ActionType):
+    def step(self, action: Optional[ActionType]):
```

Comment:
Why is the action type now optional?

Reply:
In the original version of `conversions.py`, we had the line `ActionType = Optional[int]` at the top. 

This is because we have to be able to accept `None` as a potential action, due to this line: https://github.com/Farama-Foundation/PettingZoo/blob/master/pettingzoo/utils/conversions.py#L456

However, I wanted to make it more general, so I just started using `Optional[ActionType]`, so that we can use actions other than int.

Reply:
Ahh okay that makes sense. Sounds good, good catch.